test/uk/gov/hmrc/trusts/controllers/TrustVariationsControllerSpec.scala

```diff
@@ -0,0 +1,332 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.trusts.controllers
+
+import java.util.UUID
+
+import org.mockito.Matchers._
+import org.mockito.Matchers.{eq => Meq}
+import org.mockito.Mockito._
+import org.scalatest.BeforeAndAfter
+import play.api.libs.json.Json
+import play.api.test.Helpers._
+import uk.gov.hmrc.auth.core.AffinityGroup.Organisation
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.audit.http.connector.AuditConnector
+import uk.gov.hmrc.trusts.BaseSpec
+import uk.gov.hmrc.trusts.config.AppConfig
+import uk.gov.hmrc.trusts.controllers.actions.FakeIdentifierAction
+import uk.gov.hmrc.trusts.exceptions._
+import uk.gov.hmrc.trusts.models.variation.{TrustVariation, VariationResponse}
+import uk.gov.hmrc.trusts.services.{AuditService, DesService}
+import uk.gov.hmrc.trusts.utils.Headers
+
+import scala.concurrent.Future
+
+class TrustVariationsControllerSpec extends BaseSpec with BeforeAndAfter {
```

Comment:
TrustsVariationsController specs do not reflect what is in master.
We need to add back in the validation service.

Reply:
complete